introduction/de/02-problems-solved.asciidoc

```diff
@@ -0,0 +1,38 @@
+
+. *Abwarten.* Das konsumierende Team kann nichts tun und ohne die angeforderte Funktionalität nur bedingt Fortschritte erzielen. 
+Diese Option erfordert den geringsten Arbeitsaufwand auf ihrer Seite als Konsumer. 
+Abhängig vom Nutzen der Funktionsanforderung kann das Warten in Ordnung sein. 
+Es kann jedoch zu erheblichen Schwierigkeiten kommen, insbesondere wenn die angeforderte Funktion nie geliefert wird.
+
+. *Problemumgehung.* Ein konsumierendes Team das nicht warten möchte, kann an einer anderen Stelle zusätzliche Arbeit leisten, um das Fehlen der angeforderten Funktion zu kompensieren. 
+Diese zusätzliche Arbeit schlägt sich typischerweise als Änderung im konsumierenden Projekt nieder. 
+Alternativ können Sie ein neues Projekt erstellen das ihren Anforderungen entspricht und die Verwendung des gesamten oder eines Teils des Codes des Produktionsteams ersetzt (Code- / Projektduplizierung). 
+Diese Strategie ermöglicht es dem konsumierenden Team die angeforderte Funktion aus eigener Kraft zu erhalten. Dies hat jedoch mehrere Nachteile.
+.. Alle Veränderungen die vom Verbraucherteam ausgeführt werden, stehen anderen Verbrauchern mit derselben Funktionsanforderung nicht zur Verfügung.
+.. Das konsumierende Team hat sich unbeabsichtigt darauf eingelassen die langfristige Pflege des neu geschriebenen Codes zu uebernehmen obwohl die Thematik ausserhalb der Kernteamkompetenz des Teams liegt.
+.. Das Unternehmen bekommt im Laufe der Zeit doppelte Projekte und Code im selben Problembereich.
+
+*Eskalation.* Das konsumierende Team nimmt möglicherweise kein "Nein" als Antwort hin und wendet sich stattdessen an jemanden in der Managementhierarchie der Produzenten um das produzierende Team zu beeinflussen (oder zu zwingen), die gewünschte Arbeit zu erledigen. 
+Diese Option klingt für das konsumierende Team attraktiv da es die angeforderte Funktion erhält ohne die Arbeit zur Implementierung oder Wartung zu erfordern. 
+Es ist jedoch immernoch eine Belastung für das Team, da es notwendigerweise einen Teil ihrer Aufmerksamkeit und Arbeit auf die nicht-technische Aufgabe der Eskalation lenkt. 
+Darüber hinaus lässt sich diese Option nicht skalieren da ein Verbraucher nur so oft Feature-Anfragen eskalieren kann, bevor seine Glaubwürdigkeit beeinträchtigt wird. 
+Die Eskalation ist für das Produktionsteam ähnlich störend (umso mehr) da es aus seinen normalen Workflow- und Priorisierungsmethoden herausgenommen wird um die eskalierte Funktionsanforderung zu bearbeiten.
+
+Diese Diskussion bereitet die Bühne für InnerSource. 
+InnerSource ist eine gute Lösung für die oben beschriebene Situation in der ein konsumierendes Team nicht in der Lage ist, die Funktionsanforderungen selbst zu erfüllen. 
+InnerSource bietet den Teams die Vorteile der oben beschriebenen Strategien des _Abwartens_, der _Problemumgehung_ und der _Eskalation_, ohne die damit verbundenen Nachteile.
```

Comment:
This is likely a change we need to make in the original English text: In the paragraph where the three solutions "wait it out, workaround and escalate" are being described we do not talk about any advantages but only disadvantages. So referring to advantages kept from all three approaches IMHO doesn't make a lot of sense.

Reply:
True! For the translation for now I suggest to keep the alignment. In a new step we may need to review the English text and adjust all downstream translations,